packages/shared/src/modules/routes/libs/types/routes-find-all-options.type.ts

```diff
@@ -1,5 +1,6 @@
 type RoutesFindAllOptions = {
-	name?: string;
+	categories?: string[] | undefined;
+	name?: string | undefined;
```

Comment:
Why do we need to set undefined here? We should either have those query params, or do not have categories/name key query param at all

```suggestion
	categories?: string[];
	name?: string;
```